imports/plugins/core/orders/client/templates/workflow/cancelOrder.js

```diff
@@ -0,0 +1,131 @@
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+
+const validateComment = (comment) => {
+  check(comment, Match.OptionalOrNull(String));
+  // Valid
+  if (comment.length >= 10) {
+    return true;
```

Comment:
I think you should return empty object here `{}`
Why?  See code where `validateComment` is used below